### PR TITLE
Update console file content in docs

### PR DESCRIPTION
**Pull request by Rixafy**
File console was updated to Nette 3 and Application::run() is no longer in exit() function.

### CODE REVIEW

**Comment by f3l1x** [comment]
Hold your 🏇🏇🏇

We need to think about it. There should be 2 variants for Nette 2.4 and maybe for Nette 3.0, because people don't upgrade immediately.

**Comment by Rixafy** [comment]
So, extension is still compatible with nette/di 2.4? 🤔

**Comment by mabar** [comment]
It is not about nette/di. It is about nette/web-project. Most of the people who upgrade from 2.4 or lower to 3.0 will simply don't have App\Booting class, so your PR is applicable only for new projects.

**Comment by Rixafy** [comment]
It's not depended on Booting class, thats the point, previous version is depended on bootstrap.php, now it's via composer autoload

File: .docs/README.md [hunk 1]
```diff
@@ -149,12 +149,12 @@ Make sure to set it as executable. `chmod +x <root>/bin/console`.
...
-$container = require __DIR__ . '/../app/bootstrap.php';
+declare(strict_types=1);
 
-// Get application from DI container.
-$application = $container->getByType(Contributte\Console\Application::class);
+require __DIR__ . '/../vendor/autoload.php';
 
-// Run application.
-exit($application->run());
+App\Booting::boot()
```
Comment (mabar) [inline .docs/README.md]: @Rixafy It is depending on App\Booting.

**Comment by mabar** [comment]
From my point of view, we need to explain user that he needs a Nette\DI\Container

**Comment by Rixafy** [comment]
Or we can search for Booting class first, then bootstrap.php

**Comment by Rixafy** [comment]
Booting class should have some Boot interface, because there are some people who rather rename the class to Kernel or something else, we shouldn't rely on fact that this class will always exist

**Comment by mabar** [comment]
We shouldn't rely even on interface existence. It's a userland code. If it would be possible to rely on interface existence then binaries could be in vendor instead of docs.

**Comment by Rixafy** [comment]
Well, it's only docs, we could have one version for nette 3 and another for nette 2.4, I can add the old file content somewhere under (as soon as I get home). Interface should be somewhere in nette vendor, then file console could be also in this repo, and if user wants bin/console executable, he can write his own or create link to contributte vendor.

**Comment by JanTvrdik** [comment]
Are you sure this will not break exit codes? It's extremely important that console commands return proper exit code under all circumstances.

**Comment by Rixafy** [comment]
Can we test it? I think code without exit at the end is basically the same code, can't return exit code 1, if we can somehow find out exit code.

**Comment by Rixafy** [comment]
Ok, tested using
```sh
#!/bin/bash

output=`php console test:test`
exitcode=$?

if [ "$exitcode" = "0" ]; then
   echo "Success"
fi
if [ "$exitcode" = "1" ]; then
   echo "Failure "
fi
```
and without `exit` method it still returns 0, so it should be ok.

**Comment by JanTvrdik** [comment]
> and without exit method it still returns 0, so it should be ok.

The problem is the inverse scenario. You need to return non-zero code when `Command::execute()` returns e.g. 123

**Comment by Rixafy** [comment]
I thought that run() returns void, thx, fixed.

File: .docs/README.md [hunk 2]
```diff
@@ -145,6 +145,24 @@ You can copy & paste it to your project, for example to `<root>/bin/console`.
 
 Make sure to set it as executable. `chmod +x <root>/bin/console`.
 
+##### If you are using Nette v3.x, use this script
```
Comment (f3l1x) [inline .docs/README.md]: Nette 3.0+

File: .docs/README.md [hunk 3]
```diff
@@ -145,6 +145,24 @@ You can copy & paste it to your project, for example to `<root>/bin/console`.
...
+
+require __DIR__ . '/../vendor/autoload.php';
+
+exit(App\Bootstrap::boot()
+    ->createContainer()
+    ->getByType(Contributte\Console\Application::class)
+    ->run());
+```
+
+##### Use this script if you are using Nette v2.4 or lower
```
Comment (f3l1x) [inline .docs/README.md]: Nette <=2.4

**Pull request merged**